media-sound/ladish compilation failure, this statement may fall through [-Werror=implicit-fallthrough=]

`media-sound/ladish-9999` currently fails to build with the following error:

```
20:07:47 runner ['x86_64-pc-linux-gnu-gcc', '-O2', '-pipe', '-fvisibility=hidden', '-Wall', '-Wextra', '-Werror', '-I/var/tmp/portage/media-sound/ladish-9999/work/ladish-9999/build', '-I/usr/include/dbus-1.0', '-I/usr/lib64/dbus-1.0/include', '-I/usr/include/uuid', '-DHAVE_CONFIG_H', '../daemon/escape.c', '-c', '-o', 'daemon/escape.c.1.o']
../daemon/escape.c: In function 'escape':
../daemon/escape.c:49:10: error: this statement may fall through [-Werror=implicit-fallthrough=]
       if ((flags & LADISH_ESCAPE_FLAG_OTHER) == 0)
          ^
../daemon/escape.c:53:5: note: here
     case '<':               /* invalid attribute value char (XML spec) */
     ^~~~
cc1: all warnings being treated as errors
```

--- a/media-sound/ladish/ladish-1.ebuild
+++ b/media-sound/ladish/ladish-1.ebuild
@@ -56,6 +56,7 @@ PATCHES=(
 
 src_prepare()
 {
+	sed -i -e "s/RELEASE = False/RELEASE = True/" wscript
 	append-cxxflags '-std=c++11'
 	default
 }

--- a/media-sound/ladish/ladish-9999.ebuild
+++ b/media-sound/ladish/ladish-9999.ebuild
@@ -54,6 +54,7 @@ PATCHES=(
 
 src_prepare()
 {
+	sed -i -e "s/RELEASE = False/RELEASE = True/" wscript
 	append-cxxflags '-std=c++11'
 	default
 }

